addons/mail/static/src/messaging/entity/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -98,16 +98,23 @@ function MessagingNotificationHandlerFactory({ Entity }) {
         async _handleNotificationChannel(data) {
             const {
                 channelId,
+                id,
                 info,
                 is_typing,
                 last_message_id,
                 partner_id,
             } = data;
             switch (info) {
                 case 'channel_fetched':
-                    return; // disabled seen notification feature
+                    return this._handleNotificationChannelFetched({
+                        id,
```

Comment:
Yeah but this makes insert nearly impossible or really ugly... because in the function `_findFunctionFromData` you get insert commands and it's really bad I think to use this to find an existing record

Reply:
Yeah... I've been struggling for quite a long time to find a way to "normalize" data from relational field to express this...

Still haven't found the solution... Maybe it's about marking a relational field as part of identity of current entity, something alike to foreign key in DBMS?

Anyway, too complicated to add this right now, especially when we don't have the notion of "required" fields and the technical challenges to ensure such required fields are always defined (from creation, update, computed fields, etc...), and prohibit updates on these fields...

Reply:
So I keep this id leak for now ?

Reply:
I'm afraid I don't have a better solution right now...